operators/pkg/forge/services.go

```diff
@@ -35,6 +35,9 @@ const (
 	GUIPortName = "gui"
 	// MyDrivePortName -> the name of the port the "MyDrive" service is exposed to.
 	MyDrivePortName = "mydrive"
+
+	// XVncPort -> the port in the container in which the X server is accessible through VNC.
+	XVncPort = int32(5900)
```

Comment:
Would be consistent with the others. One entry terminating with `Number` for the port number, and one with `Name` for the port name (would also keep the comments consistent).

Reply:
thing is we don't actually have a service for vnc itself, should I still make a port name?

Reply:
Also in the deployment the ports should be named